liberty-maven-plugin/src/it/ear-project-it/SampleWLP/src/test/java/net/wasdev/wlp/maven/test/it/LooseConfigTestIT.java

```diff
@@ -81,12 +81,11 @@ public void testLooseApplicationFileContent() throws Exception {
         
         expression = "/archive/archive/file";
         nodes = (NodeList) xPath.compile(expression).evaluate(inputDoc, XPathConstants.NODESET);
-        // Note: log4j-core also pulls in log4j-api
-        assertEquals("Number of <archive/> element ==>", 7, nodes.getLength());
-        // test runtime scope dependency to be incldued in the ?WEB-INF/lib
-        assertEquals("file targetInArchive attribute value", "/WEB-INF/lib/log4j-core-2.15.0.jar", 
+        assertEquals("Number of <archive/> element ==>", 5, nodes.getLength());
```

Comment:
How did it go from 7 to 5? Seems like it should have gone from 7 to 6. Was commons-io already included in the file even though the projects did not explicitly pull it in?

Reply:
Originally, the log4j-core and log4j-api showed up twice (in web.war and SampleWAR2.war). With commons-io showing up twice instead, the total number dropped from 7 to 5.